tests/test_req_token.py

```diff
@@ -39,8 +42,9 @@ def setUpClass(cls):
         environ["SAP_EXT_JWT_ALG"] = "*"
 
         cls.DEVNULL = open(devnull, 'w')
-        cls.flask_process = Popen(['flask', 'run', '-p', flask_port, '-h', 'localhost'],
-                                  env=flask_env, stdout=cls.DEVNULL, stderr=cls.DEVNULL)
+        cls.flask_process = Popen(['/sapmnt/home/I506278/git/github/sap/cloud-pysec/.venv/bin/flask',
```

Comment:
tests are failing because of this I believe

Reply:
Yes, I added that in order to enable debugging in my editor. The `flask` executable is not on PATH, it's just  sitting inside a virtualenv for me. I've fixed it now.

We could start the "mock app" in-process, avoiding the need for `Popen` - just a thought.